core/server/api/canary/utils/serializers/input/posts.js

```diff
@@ -183,6 +183,9 @@ module.exports = {
             });
         }
 
+        delete frame.data.posts[0].comment_id;
```

Comment:
`"strip": true` should do the trick here ;) no need to have code duplication

Reply:
but please double check as I'm speaking from the top of the head :)

Reply:
@gargol Double checked, it does remove it from data with `strip` keyword 👍